mozilla/kuma
# Bug 1197444: Move stylesheets, JS to kuma/static

Setup:

Be sure to do a `vagrant provision` before testing this.

Some general testing guidelines:

* Verify that the site looks and behaves as expected
* Verify that Intern tests pass
* Verify that the above again while emulating production ([instructions](https://kuma.readthedocs.org/en/latest/development.html#production-assets))

Other notes:

In the past, running `./manage.py compress_assets` was enough to enable syntax highlighting. Because minified and non-minified assets are now hosted very differently, we need to fully enter production mode ([instructions](https://kuma.readthedocs.org/en/latest/development.html#production-assets)) to enable syntax highlighting.

Once we switch to django-pipeline, we may be able to get syntax highlighting to work in both environments without any extra work, which would be an improvement even over what we have now.
